src/main/java/com/softteam/vocabuilder/service/CategoryServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.softteam.vocabuilder.service;
+
+import com.softteam.vocabuilder.persistence.entity.Category;
+import com.softteam.vocabuilder.persistence.repository.CategoryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class CategoryServiceImpl implements ICategoryService{
+    @Autowired
+    private CategoryRepository categoryRepository;
+    @Override
+    public Category create(Category category) {
+        return categoryRepository.save(category);
+    }
+
+    @Override
+    public void update(Category category) {
+        System.out.println("Service");
+        categoryRepository.save(category);
+    }
+
+    @Override
+    public Optional<Category> read(Long id) {
+        return categoryRepository.findById(id);
+    }
+
+    @Override
+    public void delete(Long id) {
+        Optional<Category> optionalCategory = categoryRepository.findById(id);
+        if(optionalCategory.isEmpty()){
+            //thorw new
+        }
```

Comment:
why this empty block?

Reply:
Because I didn't know if I had to implement what was related to validations, but that was already made clear by slack.
I've already moved to the VOC-2-create-crud-for-categories ticket to do the validations.